src/Sulu/Bundle/CoreBundle/Build/CacheBuilder.php

```diff
@@ -1,48 +0,0 @@
-<?php
```

Comment:
this is needed because this wont work inside the builder bundle and destroys the container.

Reply:
Is this still true with the fix we've built into this? I.e. skipping if we have Symfony 3.4?

Reply:
because of the deprecation notice `and will be removed in 2.0` yes it is true :)